exchange/bidder.go

```diff
@@ -113,7 +114,14 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 		}
 
 		if httpInfo.err == nil {
-			bids, moreErrs := bidder.Bidder.MakeBids(request, httpInfo.response)
+			var bidReq *openrtb.BidRequest
+			err := json.Unmarshal(httpInfo.request.Body, &bidReq)
+			if err != nil {
+				errs = append(errs, err)
+				continue
```

Comment:
I think this should go into `MakeRequests` method in rubicon adapter and not return a corresponding RequestData object if this fails.